impr(quotes): add French quotes (Augustin68)

Augustin68 opened the pull request:
### Description

Hi, after tried a PR last year I come back with some french quotes.
I hope this time they are filtered enough to not have bad words :)

Here is the list of translated quotes :)
[PR_message.md](https://github.com/user-attachments/files/15506703/PR_message.md)

monkeytypegeorge commented:
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9310546876

monkeytypegeorge commented:
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9331724959

Miodec commented:
619 seems to be using vulgar language.

Miodec commented on frontend/static/quotes/french.json:
```diff
@@ -3662,9 +3662,9 @@
-      "text": "Il a pas dit bonjours on a du lui niquer sa mère il a osé j'en ai abusé Il a pas dit bonjours on a du lui niquer sa mère il a osé j'en ai abusé Il a pas dit bonjours on a du lui niquer sa mère il a osé j'en ai abusé Il a pas dit bonjours on a du lui niquer sa mère il a osé j'en ai abusé",
-      "source": "Valde de grenoble - Il a pas dit bonjour",
-      "length": 287,
+      "text": "Voyager jusqu'au Nirvana, Hôtel cinq étoiles prendre le p'tit déj' en pyjama, Faire un p'tit pétou sur le Mont Fujiyama, Faire le tour d'la Thaïlande en 500 Yamaha (okay)",
+      "source": "Naps - La kiffance",
+      "length": 170,
```
Seems like punctuation is missing here.

Augustin68 replied:
Done.

Translation 
619 : Travel to Nirvana, Five-star hotel, have breakfast in your pajamas, Take a little splash on Mount Fujiyama, Tour Thailand in a 500 Yamaha (okay)

968 : Tremble with fear! Tremble with joy! She's coming, she's here! Tremble with fear! Tremble with joy! Because here is Karaba!

Miodec commented on frontend/static/quotes/french.json:
`Ha ha ha ha ha ha ha ha`? Probably remove

Miodec commented on frontend/static/quotes/french.json:
Its just the same word over and over again

Miodec commented on frontend/static/quotes/french.json:
I keep seeing these `(...)` from time to time. Remove them.

Miodec commented on frontend/static/quotes/french.json:
This one seems to include the source in the actual text.

Miodec reviewed (changes requested):
Check all quotes again for gramatical errors, low quality quotes and adult content.

monkeytypegeorge commented:
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9422484758

Augustin68 commented:
> Check all quotes again for gramatical errors, low quality quotes and adult content.

Gone trough all the quotes checking for bad or strange quotes, adult content or badly placed sources. I have replaced some of them with new ones.
Also removed all (...) and done the requested changes.
